plugin-server/src/ingestion/ai-costs/providers/manual-providers.ts

```diff
@@ -67,6 +67,20 @@ export const manualCosts: ModelRow[] = [
             completion_token: 0.000000075,
         },
     },
+    {
+        model: 'gemini-2.5-pro-preview',
+        cost: {
+            prompt_token: 0.00000125,
```

Comment:
I know this is a kind of quick fix, but what if you instead did something like this instead, and handled in the function where we get the price

```suggestion
            prompt_token: { 0: 0.00000125, 200000: 0.0000025 },
```

I believe the code would actually look slightly simpler

Reply:
this is a very strange case with gemini and doesnt align with how we manage model variants. this would end up with 99% of models having prompt_token { 0: price } which is more messy I think

Reply:
So to handle thinking ect we add a new model in the costs with a ":" then the modification suffix

Reply:
I would keep the existing behavior, implicit `{ 0: 0.00... }` if you had just a number in there. I dont think the code would get that complicated tbh, but if you're happy with this implementation, Im happy :)